src/ui/panel.ts

```diff
@@ -110,7 +110,7 @@ namespace Panel {
    * An options object for creating a panel.
    */
   export
-  interface IOptions {
+  interface IOptions extends Widget.IOptions {
```

Comment:
As a policy, I don't extend other classes options. I just repeat what I want out of them. This prevents a case in the future where the base class make take some option, but the subclass is relying on the option to not be configurable. If you extend the base options, it might slip through the cracks.